firefox-ios/Client/Frontend/Home/HomepageViewModel.swift

```diff
@@ -198,7 +199,9 @@ class HomepageViewModel: FeatureFlaggable, InjectedThemeUUIDIdentifiable {
         guard !viewAppeared else { return }
 
         viewAppeared = true
-        Experiments.events.recordEvent(BehavioralTargetingEvent.homepageViewed)
+        throttler.throttle {
```

Comment:
Nit: would it make sense to add a `ToDo` here with the Jira you filed (FXIOS-9428)? Since addressing that should allow us to remove this throttler in the future

Reply:
good point! will add